src/kontrol/prove.py

```diff
@@ -413,7 +413,8 @@ def _method_to_cfg(
         new_node_ids = [init_node.id]
         init_node_id = init_node.id
 
-    is_test = method.signature.startswith('test')
+    proof_prefixes = ['test', 'proof', 'check']
```

Comment:
```suggestion
    proof_prefixes = ['test', 'prove', 'check']
```

Reply:
By the way, why support both `prove` and `check`? If there's no specific reason, I think we should just pick one of the two.

Reply:
So we can automatically verify tests written for other symbolic testing tools: `hevm` (at least, the last time I checked) symbolically executes tests starting with `prove`, and `halmos` — tests starting with `check`.

Reply:
Thanks for noticing the wording issue! Fixed "prove" spelling in https://github.com/runtimeverification/kontrol/pull/249/commits/0b4e7357e6df8f1a089bdf779774b653cbc8734f.